tests/virt_autotest/install_package.pm

```diff
@@ -54,41 +55,19 @@ sub install_package {
     }
 
     if ($dependency_repo) {
-        if (check_var('ARCH', 's390x')) {
-            lpar_cmd("zypper --non-interactive --no-gpg-check ar -f ${dependency_repo} dependency_repo");
-            lpar_cmd("zypper --non-interactive --gpg-auto-import-keys ref");
-            lpar_cmd("zypper --non-interactive in $dependency_rpms");
-            lpar_cmd("zypper --non-interactive rr dependency_repo");
-        }
-        else {
-            zypper_call("--no-gpg-check ar -f ${dependency_repo} dependency_repo");
-            zypper_call("--gpg-auto-import-keys ref", 180);
-            zypper_call("in $dependency_rpms");
-            zypper_call("rr dependency_repo");
-        }
+        zypper_wrapper("--no-gpg-check ar -f $dependency_repo dependency_repo");
```

Comment:
Although x86 and s390 cmds are similar, but they are with intended slight changes, that's why @guoxuguang add branches for s390. Please correct me if I am wrong.